pkg/graph/resolvers/trb_request.go

```diff
@@ -414,11 +414,11 @@ func ReopenTRBRequest(
 }
 
 // IsRecentTRBRequest determines if a TRB Request should be determined to be flagged as "recent" or not.
-// TODO: Add more logic in https://jiraent.cms.gov/browse/EASI-2711
 func IsRecentTRBRequest(ctx context.Context, obj *models.TRBRequest, now time.Time) bool {
 	numDaysToConsiderRecent := -7
 	recentIfAfterDate := now.AddDate(0, 0, numDaysToConsiderRecent)
-	return obj.CreatedAt.After(recentIfAfterDate)
+	isRecent := obj.State != models.TRBRequestStateClosed && (obj.CreatedAt.After(recentIfAfterDate) || obj.TRBLead != nil)
```

Comment:
I might also recommend breaking up these bits into their own `bool` values for clarity, i.e.

```go
notClosed := obj.State != models.TRBRequestStateClosed
noLeadAssigned := obj.TRBLead != nil
```

that way you can have a line that reads like
```go
isRecent := notClosed && (isAfterDate || noLeadAssigned)
```

Reply:
e74bb3d4b